odoo/addons/base/tests/test_reports.py

```diff
@@ -455,24 +455,45 @@ def test_pdf_render_page_overflow(self):
         pdf_content = self.create_pdf(page_content=page_content)
         pages = self._parse_pdf(pdf_content)
 
-        self.assertEqual(len(pages), 4, '4 pages are expected, 2 per record (you may ensure `nb_lines` has a correct value to generate an oveflow)')
-        page_break_at = int(pages[1][2][1].split('\n')[0])  # This element should be the first line, 61 when this test was written
+        self.assertEqual(len(pages), 6,
+                        '6 pages are expected, 3 per record (you may ensure `nb_lines` has a correct value to generate an oveflow)')
+        first_page_break_at = int(
+            pages[1][2][1].split('\n')[0])  # This element should be the first line, 61 when this test was written
+        second_page_break_at = int(pages[2][2][1].split('\n')[0])
```

Comment:
how do you explain there's one more page per record, now? :thinking: 

and is `first_page_break_at` still `61`?? :suspect: